QuantumToolbox.jl/time_evolution/Dicke.qmd

```diff
@@ -164,20 +168,19 @@ slists = []
 Ns = 8:8:24
 
 for N in Ns
- slist = []
+    slist = []
     for g in gs
- H_ = H(M0, N, g)
- _, vecs = eigenstates(H_)
- S = entropy_mutual(vecs[1], 1, 2)
+        H_ = H(M0, N, g)
+        _, vecs = eigenstates(H_)
+        S = entropy_mutual(vecs[1], 1, 2)
         push!(slist, S)
     end
-
     push!(slists, slist)
 end
```

Comment:
I see it only now. Also this case is inefficient and should be avoided in the tutorials. We are initializing an empty vector `slist = []` of type `Any`. Moreover, I always recommend to not use `push!`, as it allocates memory.

You could use the same example I suggested before, with `slist = map(gs) do g`.